demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -88,9 +88,11 @@ def execute_update(self):
             self.check_rn_dir(rn_path)
             changed_files = {}
             self.find_added_pack_files()
+            is_docker_image_changed = False
             for packfile in self.modified_files_in_pack:
                 file_name, file_type = self.identify_changed_file_type(packfile)
-
+                if 'yml' in packfile:
```

Comment:
This should be done by `file_type` - that way you won't check playbooks

Reply:
good idea.
I'll add it.